src/Footer.jsx

```diff
@@ -20,6 +20,8 @@ import withMatchMedia from './utils/components/withMatchMedia';
 
 import { getSocialLinks } from './utils/getSocialLinks';
 
+export const defaultLegalCopy = 'Meetup is a wholly owned subsidiary of WeWork Companies, Inc.';
```

Comment:
Is this the right way to go in this repo -- english defaults here and 'apps' pass in translated strings?

Reply:
This is fine, but I'd recommend just defining this in the story and passing it in from there.

Reply:
Hey @mperrotti -- just getting back today. So is the recommendation to define the english string in the story and the component separately? Is it okay that the story and component might get out of sync copy-wise. I'm fine to change it if that's how it's done here, just trying to understand the reasoning.

Reply:
I'm suggesting not to define anything in the component. Just put the `legalCopy` prop in place, and then pass `legalCopy="Meetup is a wholly...`